float64[7] qc2
float64[7] qc1
float64[7] tau_d
float64[7] term1
float64[7] term2
float64[7] term3
float64[7] term4
float64[6] F_estimated
#
float64[7] jointError
float64[6] cartesianError
#
float64[7] q
float64[7] q_d